docs/resources/group.md

```diff
@@ -106,6 +106,7 @@ resource "azuread_group" "example" {
 
 The following arguments are supported:
 
+* `administrative_unit_id` - (Optional) The administrative unit ID that the group is member of.
```

Comment:
We should be clearer about why this property is here and nudge the practitioner towards the more flexible `azuread_administrative_unit_member` resource or the `members` property of `azuread_administrative_unit`, and use this property only if they need to create a single AU scoped group.
```suggestion
* `administrative_unit_id` - (Optional) The object ID of a single administrative unit in which the group is a member. If specified, new groups will be created in the scope of the administrative unit. If empty, new groups will be created at the tenant level.

!> **Warning** Do not use the members property at the same time as the [azuread_administrative_unit_member](https://registry.terraform.io/providers/hashicorp/azuread/latest/docs/resources/administrative_unit_member) resource, or the `members` property of the [azuread_administrative_unit](https://registry.terraform.io/providers/hashicorp/azuread/latest/docs/resources/administrative_unit#members) resource, _for the same group_. Doing so will cause a conflict and administrative unit members will be removed.
```